application/src/main/scala/com/azavea/franklin/database/Page.scala

```diff
@@ -0,0 +1,20 @@
+package com.azavea.franklin.database
+
+import io.circe.generic.semiauto._
+import scala.util.Try
+
+final case class Page(limit: Option[Int], next: Option[String]) {
+  lazy val pageInt: Option[Int] = next.flatMap(v => Try(v.toInt).toOption)
+
+  lazy val offset: Int = pageInt match {
+    case Some(offsetInt) => offsetInt * limit.getOrElse(20)
+    case _               => 0
+  }
+
+  lazy val nextPage = Page(limit, pageInt.map(v => s"${v + 1}"))
```

Comment:
Since this isn't used anywhere yet, I think we should maybe avoid locking down the API. It's not clear whether we're supposed to support token-based paging a la S3 queries or page numbers or what, so we should leave it open to future us who has that information

Reply:
I think the interpretation and use of next is left open to specific implementations -- in this case I'm treating it as a page number that can be used to compute the offset based on a given limit.

I don't think this needs to be our _final_ implementation, but is at least a start. I looked into _why_ it wasn't being included and it turns out that I messed up in the PR and it actually does need to be used. This is resolved by https://github.com/azavea/franklin/pull/29/commits/80cd4c657b6665e6d4b593647cebcf33bade0c7e -- the feature collection returned [needs](https://github.com/radiantearth/stac-spec/tree/master/api-spec/extensions/search#itemcollection-fields) to have a top level property "search:metadata" with information on `next`, `limit`, `matched`, and `returned`. The PR is updated to reflect that requirement and deletes the other feature collection being returned.

That being said, I certainly would prefer a better paging system - a token-based approach would certainly qualify, but I also think that deserves some additional thought -- maybe its own ADR -- since I rather us have a plan. I opened https://github.com/azavea/franklin/issues/34 for this.